src/python/pcmdi/scripts/pcmdi_metrics_driver.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/data/igcmg/database/PMP_install/PCMDI_METRICS/bin/python
```

Comment:
@jservonnat do you want to submit a revised PR (so just commit the edit/removal of the custom path to your existing branch), that way it's a simple approve, rather than having to clone, edit and then merge it